fireboard/README.md

```diff
@@ -0,0 +1,144 @@
+## Fireboard Monitoring Template
+
+Provided by: [Scott Anderson](https://github.com/sanderson/)
+
+Use this InfluxDB template to monitor data collected by the
+[Fireboard cloud-connected thermometer](https://www.fireboard.com/).
+
+![Fireboard Dashboard Screenshot](img/fireboard_dashboard.png)
+
+## Quick Install
+
+### InfluxDB UI
+
+In the InfluxDB UI, go to **Settings > Templates** and enter the following URL:
+
+```sh
+https://raw.githubusercontent.com/influxdata/community-templates/master/fireboard/fireboard.yml
+```
+
+### Influx CLI
+
+If you have your InfluxDB credentials [configured in the CLI](https://docs.influxdata.com/influxdb/v2.0/reference/cli/influx/config/), install this template with:
+
+```sh
+influx apply -u https://raw.githubusercontent.com/influxdata/community-templates/master/fireboard/fireboard.yml
+```
+
+## Included Resources
+
+- 1 Bucket: `fireboard`
+- 1 Telegraf Configuration: `Fireboard`
+- 1 Dashboard: `Fireboard`
+- 12 Dashboard Variables:
+  - `probe1_target`
+  - `probe1_type`
+  - `probe2_target`
+  - `probe2_type`
+  - `probe3_target`
+  - `probe3_type`
+  - `probe4_target`
+  - `probe4_type`
+  - `probe5_target`
+  - `probe5_type`
+  - `probe6_target`
+  - `probe6_type`
+- 1 Task: `Fireboard Stall Check`
+- 1 Notification Endpoint: `Example Endpoint`
+- 1 Notification Rule: `Fireboard Stall Notification`
+- 1 Label: `Fireboard`
+
+## Setup
```

Comment:
This needs to be `## Setup Instructions` in order for the InfluxDB UI to parse this file properly.